tests/foreman/api/test_contentview.py

```diff
@@ -1722,3 +1722,33 @@ def test_positive_publish_promote_with_RH_ostree_and_other(
         content_view.publish()
         promote(content_view.read().version[0], module_lce.id)
         assert len(content_view.read().version[0].read().environment) == 2
+
+    @pytest.mark.stubbed
+    @pytest.mark.tier1
+    def test_positive_check_pulp3_is_installed(self):
+        """Ensure that pulp3 is installed
+
+        :id: 5d5690bf-56c7-4dba-bb09-de0f5f4861b1
+
+        :expectedresults: Pulp3 is active
+
+        :CaseLevel: System
+
```

Comment:
Should we add a CaseAutomation tag since these are stubbed test cases? The same goes for other stubbed tests.
```:CaseAutomation: NotAutomated```